examples/README.md

```diff
@@ -201,3 +201,30 @@ mounts of paths below /tmp will refer to the VM and not your Mac/etc
 host. See
 [this github page](https://github.com/docker/compose/issues/1039) for
 more details.
+
+### Development & Debugging
+
+When making changes to existing example containers or simply wanting to debug if
+a specific **backend** contains the data that you're expecting you will have the
+need to connect to said **backend**. You can easily see which containers are
```

Comment:
Also, how about a sentence mentioning that the docker-compose files expose all the underlying data sources with randomly assigned host ports? That would set up why the docker-compose ps is necessary.